CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+ # Changelog
+
+All notable changes to this project will be documented in this file.
+
+## [Unreleased]
+
+### Added 
+- Add the initial Xcode project.
+- Add the Swiftlint script to the build phase of the Woof target.
+- Add the Swiftformat script to the build phase of the Woof target.
+- Update the deployment target to 15.0. 
+
+## [0.1.0]
```

Comment:
Do you have the version `0.1.0` already?
I thought you will change the line "Unreleased" to "0.1.0" when the times come. Am I right?

Reply:
Yes, agree.  Fixed it.